src/components/Navbar.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from "next/link";
+import Image from "next/image";
+import { useState } from "react";
+import logo from "../assets/stellar-xlm-logo-full.svg";
+import { Avatar } from "antd";
+import { FaRegMoon } from "react-icons/fa";
+import { FiSun } from "react-icons/fi";
+import { useEffect } from "react";
+
+const Navbar = () => {
+  const [theme, setTheme] = useState<string>("light");
+
+  const handleChangeTheme = () =>
+    setTheme((prevTheme) => (prevTheme === "light" ? "dark" : "light"));
+
+  useEffect(() => {
+    console.log(theme);
```

Comment:
Remove this, please.

Reply:
Solved in commit 0109e918ef7b246d5657c1ca036c4711e157b3c8.